tests/move_architecture/test_architecture.py

```diff
@@ -0,0 +1,107 @@
+# Copyright 2024 Qiskit on IQM developers
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Testing extended quantum architecture specification
+"""
+from typing import Union
+
+from qiskit.circuit import Instruction
+
+from iqm.iqm_client import QuantumArchitecture, QuantumArchitectureSpecification
+from tests.move_architecture.move_architecture import move_architecture_json, move_architecture_specification
+from tests.utils import get_mocked_backend
+
+
+def test_parse_move_architecture_specification():
+    """Tests that the extended architecture specification can be parsed"""
+    arch = QuantumArchitectureSpecification(**move_architecture_specification)
+    check_arch(arch)
+
+
+def test_parse_move_architecture():
+    """Tests that the quantum architecture model can be parsed"""
+    arch = QuantumArchitecture(**move_architecture_json)
+    check_arch(arch.quantum_architecture)
+
+
+def check_arch(arch: QuantumArchitectureSpecification):
+    """Verifies architecture is as expected"""
```

Comment:
```suggestion
    """Verifies architecture is as expected."""
```

Reply:
And same for other new docstrings, just to be consistent with existing docstrings in the codebase.

Reply:
Fixed documentation